kolibri/utils/version.py

```diff
@@ -64,13 +70,16 @@ def derive_version_from_git_tag():
         subprocess.CalledProcessError,  # not a git repo
         OSError  # git executable doesn't exist
     ):
-        return None
+        pass
+    else:
+        version_string = parse_git_tag_version_string(git_describe_string)
 
-    return parse_git_tag_version_string(git_describe_string)
+    os.chdir(current_dir)
```

Comment:
I could... but that would require creating a context manager just for this singular instance.

http://ralsina.me/weblog/posts/BB963.html Is this anywhere in the standard library at all?